light-clients/ics08-wasm/src/instantiate.rs

```diff
@@ -1,16 +1,12 @@
-use super::client_state::ClientState;
-use crate::consensus_state::ConsensusState;
+use crate::Bytes;
 #[cfg(feature = "cosmwasm")]
 use cosmwasm_schema::cw_serde;
 
 #[cfg_attr(feature = "cosmwasm", cw_serde)]
 #[cfg_attr(not(feature = "cosmwasm"), derive(Clone, Debug, PartialEq))]
 #[derive(Eq)]
-pub struct InstantiateMessage<
-	AnyClient: Default,
-	AnyClientState: Default,
-	AnyConsensusState: Default,
-> {
-	pub client_state: ClientState<AnyClient, AnyClientState, AnyConsensusState>,
-	pub consensus_state: ConsensusState<AnyConsensusState>,
+pub struct InstantiateMessage {
+	pub client_state: Bytes,
+	pub consensus_state: Bytes,
+	pub checksum: Bytes
```

Comment:
Some additional annotations seem to be required here on the bytes fields to support the base64 string encoding of bytes in JSON.

```
diff --git a/light-clients/ics08-wasm/src/instantiate.rs b/light-clients/ics08-wasm/src/instantiate.rs
index 2ee4c00..3c7f481 100644
--- a/light-clients/ics08-wasm/src/instantiate.rs
+++ b/light-clients/ics08-wasm/src/instantiate.rs
@@ -1,4 +1,6 @@
+use crate::msg::Base64;
 use crate::Bytes;
+
 #[cfg(feature = "cosmwasm")]
 use cosmwasm_schema::cw_serde;

@@ -6,7 +8,13 @@ use cosmwasm_schema::cw_serde;
 #[cfg_attr(not(feature = "cosmwasm"), derive(Clone, Debug, PartialEq))]
 #[derive(Eq)]
 pub struct InstantiateMessage {
+       #[cfg_attr(feature = "cosmwasm", schemars(with = "String"))]
+       #[cfg_attr(feature = "cosmwasm", serde(with = "Base64", default))]
        pub client_state: Bytes,
+       #[cfg_attr(feature = "cosmwasm", schemars(with = "String"))]
+       #[cfg_attr(feature = "cosmwasm", serde(with = "Base64", default))]
        pub consensus_state: Bytes,
+       #[cfg_attr(feature = "cosmwasm", schemars(with = "String"))]
+       #[cfg_attr(feature = "cosmwasm", serde(with = "Base64", default))]
        pub checksum: Bytes
 }
```